CHANGELOG.md

```diff
@@ -1,17 +1,21 @@
 # `chainweb-node` Changelog
 
-## Unreleased
+## 1.8 (TODO)
 
 This version replaces all previous versions. Any prior version will stop working
 on **2020-04-30T00:00:00Z**. Node administrators must upgrade to this version
 before that date.
 
+This version will stop working on **2020-06-11T00:00:00Z**.
+
```

Comment:
Please add #1019, #1004, #1002

Reply:
1002 is already mentioned below. I think 1004 is not required since it's not a user visible change.

Reply:
Added